core/templates/core/post-list.html

```diff
@@ -12,26 +12,15 @@
         {% for post in posts %}
             <div class="card bg-dark shadow-sm mb-3 text-light">
                 {% include 'core/post-header.html' with post=post %}
-                <div class="card-body border post-clickable" id="post-{{ post.id }}" data-url="{% url 'post-detail' post.id %}">
-                    <h3 class="h3">{{ post.title }}</h3>
-                    {{ post.content|truncatechars:100 }}
-                </div>
+                    <a href="{% url 'post-detail' post.id %}" class="text-decoration-none">
+                        <div class="card-body text-light">
+                                <h3 class="h3">{{ post.title }}</h3>
+                                {{ post.content|truncatechars:100 }}
+                        </div>
+                    </a>
```

Comment:
Incorrect HTML. 
A can not wrap h3. It needs to be other way around
h3 > a
Div > a

```suggestion
                    
                        <div class="card-body text-light">
                                <h3 class="h3"><a href="{% url 'post-detail' post.id %}" class="text-decoration-none">{{ post.title }}</a></h3>
                                <a href="{% url 'post-detail' post.id %}" class="text-decoration-none">{{ post.content|truncatechars:100 }}</a>
                        </div>
```
You also forgot to add tags stripping to truncate. What will heppen if there will be, ie <b> not closed in those 100 chars?